### DISCUSSION TITLE
running `docusaurus start` from a codespace?

### DISCUSSION THREAD

**Discussion by pelikhan** [facebook/docusaurus#8354] [Q&A] [answered]
Hello all,

I was trying to run a new docusaurus 2.0 from a Github Codespace and it seems that the proxying is not happening. The web site opens but it can't connect to the server running on the VM. Has anyone been able to do this?

**Comment by slorber** [comment, marked as answer]
We had some issues before and received a few PRs more recently to make it work better

See also
https://github.com/facebook/docusaurus/issues/7912
https://github.com/facebook/docusaurus/issues/8306

Try 2.2, upcoming 2.3 or canary releases

**Reply by pelikhan** [reply]
Thank you for this answer! I am on 2.2 and I'll just wait for 2.3.
